### DISCUSSION TITLE
Is there any way to set a command line flag to tell git to NOT run any hooks?

### DISCUSSION THREAD

**Discussion by eamodio** [git-for-windows/git#3201] [Q&A] [answered]
I'm wondering if there is any way to tell Git to not run any hooks.

**Comment by rimrul** [comment, marked as answer]
I don't think there is a command line option for that. You might be able to achieve that result by pointing `core.hooksPath` to a non-existing or empty directory though, so something like `git -c core.hooksPath=/dev/null foo` might work.

**Reply by derrickstolee** [reply]
This won't stop hooks that are provided as fully-qualified paths, which is the secure way to do things. In the rare occasions where I have worked with hooks, I've used the full path to avoid ambiguity.

**Reply by eamodio** [reply]
Interesting -- thanks!

**Reply by dscho** [reply]
I believe the only hook for which you can configure the path is the FSMonitor hook, and it is an oddball because there is no way to "install" it by copying it to the `.git/hooks/` directory.
